src/Symfony/Component/Ldap/Tests/LdapTestCase.php

```diff
@@ -8,6 +8,14 @@ class LdapTestCase extends TestCase
 {
     protected function getLdapConfig()
     {
+        $h = @ldap_connect(getenv('LDAP_HOST'), getenv('LDAP_PORT'));
```

Comment:
I suggest checking first whether we have values for these env variable. It does not make sense to try to connect if they were not provided.

Reply:
they are in the phpunit.xml.dist so that's not really needed